ui/lib/config-ui/addon/components/messages/page/create-and-edit-message-form.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { task } from 'ember-concurrency';
+import errorMessage from 'vault/utils/error-message';
+import { inject as service } from '@ember/service';
+import { format } from 'date-fns';
+
+import { localDateTimeString } from 'vault/models/config-ui/message';
+
+/**
+ * @module Page::CreateAndEditMessageForm
+ * Page::CreateAndEditMessageForm components are used to display create and edit message form fields.
+ * @example
+ * ```js
+ * <Page::CreateAndEditMessageForm @message={{this.message}}  />
+ * ```
+ * @param {model} message - message model to pass to form components
+ */
+
+export default class MessagesList extends Component {
+  @service router;
+  @service flashMessages;
+
+  @tracked errorBanner = '';
+  @tracked modelValidations;
+  @tracked invalidFormMessage;
+  @tracked endTime;
+
+  constructor() {
+    super(...arguments);
+    if (this.args.message.endTime) {
+      this.endTime = format(new Date(this.args.message.endTime), localDateTimeString);
```

Comment:
This will clear and override the message end time on edit

Reply:
That's actually old code I forgot to remove